install/installer/trafodion_install

```diff
@@ -539,7 +543,9 @@ source $TRAF_CONFIG
 #============================================
 #Check to make sure HBase Version is the correct version
 
-checkHBaseVersion
+if [[ $HADOOP_TYPE != "apache" ]]; then
+   checkHBaseVersion
+fi
 #==============================================
```

Comment:
The instructions will provide the version you need, so I didn't include a check at this time. I will write a JIRA to add that improvement.